OrbitGl/ThreadTrack.cpp

```diff
@@ -114,13 +114,6 @@ void ThreadTrack::SetTimesliceText(const Timer& timer, double elapsed_us,
           absl::StrFormat("%s %s %s", name, extra_info.c_str(), time.c_str());
 
       text_box->SetText(text);
-    } else if (timer.m_Type == Timer::INTROSPECTION) {
```

Comment:
The introspection stuff should actually be left in, we are using it. It needs to be enabled by defining ORBIT_TRACING_ENABLED in OrbitCore/Introspection.cpp, maybe that's why you missed it?

Reply:
Yes, sorry for this, reverted.